index.html

```diff
@@ -3979,24 +3979,45 @@ <h3>Receiving and parsing content</h3>
       </li>
     </ol> <!-- parsing NDEF text record -->
   </p>
-  <p class="note" data-link-for="NDEFRecord">
-    Using the <a data-cite="encoding#encoder">encoder</a>, it is only possible
-    to encode as UTF-8, unless you do the encoding manually
-    and use the "{{opaque}}" {{recordType}}, which
-    allows you to write any opaque data.
-    <br><br>
-    When you write a <a>string</a> to the "{{text}}"
-    {{recordType}}, it will be written as UTF-8.
-    External applications have the ability to additionally encode
-    this field as UTF-16BE, but that is transparent from the use
-    of the {{NFCReader}} which will always return the data
-    as a <a>string</a>.
-    <br><br>
-    As external applications may have encoded the "{{opaque}}"
-    {{recordType}} using a different
-    encoding, which can be decoded using [[[ENCODING]]] if you know
-    the encoding.
-  </p>
+  <aside class="note" data-link-for="NDEFRecord">
+    <p>
+      The Unicode standard defines multiple encodings like UTF-8, UTF-16 and UTF-32.
+      UTF-8 is the preferred encoding on the web, and it has the advantage
+      that it is endianness agnostic, as code points are represented in single bytes.
+    </p>
+    <p>
+      The NDEF text records allow the text to be encoded as either UTF-8 and UTF-16.
+      Generally, it is preferred to use UTF-8, but integration with existing systems
+      may requires UTF-16.
+    </p>
+    <p>
+      The byte order of NDEF is big endian (BE), which means that everything is read
+      back with big endian byte ordering.
+    </p>
+    <p>
+      For UTF-16, byte order matters as it might differ between reading and writing
+      due to each code point spanning two bytes. For this reason UTF-16 encoded text
+      usually contains a byte order mark (BOM) which is written as `0xFEFF`. This means
```

Comment:
Nit: "Byte Order Mark"

Reply:
encoding writes it lower case ... let me do it differently